src/Common/src/CoreLib/System/MemoryExtensions.cs

```diff
@@ -196,6 +249,7 @@ public static int IndexOf<T>(this Span<T> span, T value)
                     ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(span)),
                     Unsafe.As<T, byte>(ref value),
                     span.Length);
+
```

Comment:
Whitespace simply makes it easier to read - it was inconsistently applied in this same pattern